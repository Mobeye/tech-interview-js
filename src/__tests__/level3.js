import * as fs from "fs";
import { level3 } from "../answers";

it("level3", () => {
  const data = JSON.parse(fs.readFileSync("./src/level3.json", "utf8"));
  expect(level3(data)).toMatchSnapshot();
});
