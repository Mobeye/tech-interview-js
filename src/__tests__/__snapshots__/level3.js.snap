// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`level3 1`] = `
Object {
  "carts": Array [
    Object {
      "id": 1,
      "total": 2350,
    },
    Object {
      "id": 2,
      "total": 1775,
    },
    Object {
      "id": 3,
      "total": 1798,
    },
    Object {
      "id": 4,
      "total": 1083,
    },
    Object {
      "id": 5,
      "total": 1328,
    },
  ],
}
`;
